llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -856,6 +855,15 @@ const SUnit *InterBlockEdges::getPreBoundaryNode(MachineInstr *MI) const {
   return &DDG.SUnits.at(Found->second);
 }
 
+const SUnit *InterBlockEdges::getPostBoundaryNode(MachineInstr *MI) const {
```

Comment:
I think this function is not used. Or do we have future plans for it?

Reply:
Good catch, I ended up not needing it. Will remove